qiime/core/archiver.py

```diff
@@ -211,12 +210,13 @@ def save(self, filepath):
                     zf.write(abspath, arcname=archive_path)
 
     def _save_version(self, zf, root_dir):
+        version = '%s\n' % self._VERSION
         zf.writestr(os.path.join(root_dir, self._VERSION_FILENAME),
-                    '%s\n' % self._VERSION)
+                    version.encode('utf-8'))
```

Comment:
Hmm, my understanding in Python 3 is that `b''` represents a bytestring literal, which would just write out the octet sequence, but maybe I am totally off target here.